### DISCUSSION TITLE
Downloading MindTheGapps fails; stops output from completing

### DISCUSSION THREAD

**Discussion by Matthew-Gentex** [LSPosed/MagiskOnWSALocal#177] [Q&A] [answered]
The script seems to properly download everything except for MindTheGapps from SourceForge. The script then finishes, but no output folder is made. Manually downloading Gapps and putting it in the downloads folder doesn't work either. Any suggestions? I'm quite new to this, so apologies if I'm doing something obviously incorrectly. 

Attached is the aria2_download.log.

[aria2_download.log](https://github.com/LSPosed/MagiskOnWSALocal/files/9780122/aria2_download.log)

**Comment by Howard20181** [comment, marked as answer]
Testing without problem. Please setup proxy or upgrade your system's ca-bundle.

**Reply by caiohamamura** [reply]
I think this should be either within script itself or at least in the documentation.

I agree that it is the user role to keep their system updated, but the regular windows/android user, which is also kind of the target for this, the error message is quite conspicuous and it is not a common knowledge the existence of certificates for https, nor that there is a package named `ca-certificates` to hold those certificates. 

Even though I was already aware about https protocol and the trusted certificates database, I didn't remember the package name before coming here.

So, again, I think it should either be in the script or the documentation, or else it is quite possible that issues will come or at least people will keep searching around for the solution.

**Reply by Howard20181** [reply]
But I don't know if the certificate package name is the same for all but the Debian distribution, and it's a pain to write down all the issues.
